client/src/js/services/PatientInvoiceService.js

```diff
@@ -120,6 +120,7 @@ function PatientInvoiceService(Modal, util, Session, Api) {
       { field: 'service_id', displayName: 'FORM.LABELS.SERVICE' },
       { field: 'user_id', displayName: 'FORM.LABELS.USER' },
       { field: 'reference', displayName: 'FORM.LABELS.REFERENCE' },
+      { field: 'debtor_uuid', displayName: 'FORM.LABELS.CLIENT' },
```

Comment:
ui-select Allows you to search for debtors in
- partials / patient_invoice / registry / search.modal.html
- partials / cash / payments / templates / search.modal.html

Should I delete it in this two Modals